From 1842c6049b546cbbca5962ae4271400b0060dc5d Mon Sep 17 00:00:00 2001
From: jerry <jerry@example.com>
Date: Mon, 24 Dec 2018 05:18:10 +0000
Subject: [PATCH]  test10.txt is commied

---
 test10.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test10.txt b/test10.txt
index e8a0fe7..f57c7dd 100644
--- a/test10.txt
+++ b/test10.txt
@@ -1 +1 @@
-how are u
+hi man how are u
-- 
2.7.4

